### PR TITLE
[IMP] add Month and Year in Relative to report base date

### LINKED ISSUE #2: new month/year relative period types
_From @sbidoul on September 24, 2017 11:9_

This is doable with date ranges, but is more intuitive if it also appears as relative period types.

WIP https://github.com/acsone/account-financial-reporting/pull/20/files

_Copied from original issue: OCA/account-financial-reporting#351_

### PR DESCRIPTION
**Pull request by sbidoul**
closes #2

### CODE REVIEW

**Comment by gonzaloruzafa** [comment]
Do you need functional review for this PR? Looks like an interesting feature to add!

**Comment by sbidoul** [comment]
Yes please, review is welcome. Are you in LLN? I'm in the front-left corner of the room.

File: mis_builder/CHANGES.rst [hunk 1]
```diff
@@ -6,6 +6,14 @@ Changelog for mis_builder
 ..
 .. *
 
+10.0.3.1.0 (unreleased)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+New features:
+
+* [ADD] month and year reletive periods, easier to use than
```
Comment (lepistone) [inline mis_builder/CHANGES.rst]: typo: `relative`

**Review by lepistone — APPROVED** [review]

**Comment by sbidoul** [comment]
Written by @Martronic-SA, review by myself and @lepistone. I merge in 10.0-dev that will become 3.1.

**Comment by Martronic-SA** [comment]
Great! Thank you!

**Pull request merged**